trunk/SUAVE/Sizing/size_from_PGM.py

```diff
@@ -0,0 +1,901 @@
+## @ingroup Sizing
```

Comment:
Can this function be used without the PGM architecture?

Reply:
Yes, potentially. We can chat tomorrow about renaming it or adding it with tutorials or something.